src/v2/api/controllers/RoleController.ts

```diff
@@ -101,7 +124,7 @@ export class RoleController {
   @ApiBadRequestResponse({
     description: `\n
     InvalidBodyException:
-      Permission can not be empty
+      Permission cannot be empty
       Set is not a boolean`,
   })
   @Post('/:roleId/grants')
```

Comment:
update the documentation for the last two endpoints (post and get '/:roleId/grants')